src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -208,6 +215,35 @@ public void removeTag(Tag t) throws TagNotFoundException {
         tags.remove(t);
     }
 
+
+    //@@author jelneo
+
+    /**
+     * Increase debts of a person by the indicated amount
+     * @param target person that borrowed more money
+     * @param amount amount that the person borrowed. Must be either a positive integer or positive number with
+     *               two decimal places
+     * @throws PersonNotFoundException if {@code target} could not be found in the list.
+     */
+    public void addDebtToPerson(ReadOnlyPerson target, Debt amount) throws PersonNotFoundException {
+        Name name = target.getName();
+        Phone phone = target.getPhone();
+        Email email = target.getEmail();
+        Address address = target.getAddress();
+        PostalCode postalCode = target.getPostalCode();
+        Debt newDebt = target.getDebt();
+        newDebt.addToDebt(amount);
+        Deadline deadline = target.getDeadline();
+        Set<Tag> tags = target.getTags();
+        Person editedPerson = new Person(name, phone, email, address, postalCode, newDebt, deadline, tags);
```

Comment:
The DateBorrow field might be affected here. The dateBorrow field is created when a new Person is created. You have to set the date from the ReadOnlyPerson target.

Reply:
fixed